core/base/ModuleManager.py

```diff
@@ -172,6 +173,11 @@ def _loadModuleList(self, moduleToLoad: str = '', isUpdate: bool = False) -> dic
 					modules[moduleInstance.name] = {
 						'instance': moduleInstance
 					}
```

Comment:
This could actually be refactored to modules[moduleName] = instance. I never understood why the list, was part of a refactor from jrk